addons/web_editor/static/src/js/editor/odoo-editor/test/spec/editor.test.js

```diff
@@ -3598,12 +3598,259 @@ X[]
+                                            '<td>ab</td>' +
+                                            '<td>cd[]</td>' +
+                                            '<td>ef</td>' +
+                                        '</tr>' +
+                                        '<tr style="height: 40px;">' +
+                                            '<td>ab</td>' +
+                                            '<td>cd</td>' +
+                                            '<td>ef</td>' +
+                                        '</tr></tbody></table>',
+                            stepFunction: async editor => editor.execCommand('addColumnRight'),
+                            contentAfter: '<table style="width: 200px;"><tbody><tr style="height: 20px;">' +
+                                            '<td style="width: 38px;">ab</td>' +
+                                            '<td style="width: 50px;">cd</td>' +
+                                            '<td style="width: 50px;"><br></td>' +
+                                            '<td style="width: 61px;">ef</td>' +
+                                        '</tr>' +
+                                        '<tr style="height: 30px;">' +
+                                            '<td>ab</td>' +
+                                            '<td>cd[]</td>' +
+                                            '<td><br></td>' +
+                                            '<td>ef</td>' +
+                                        '</tr>' +
+                                        '<tr style="height: 40px;">' +
+                                            '<td>ab</td>' +
+                                            '<td>cd</td>' +
+                                            '<td><br></td>' +
+                                            '<td>ef</td>' +
+                                        '</tr></tbody></table>',
+                        });
+                    });
+                });
+            });
+        });
         describe('tab', () => {
             it('should add a new row on press tab at the end of a table', async () => {
                 await testEditor(BasicEditor, {
-                    contentBefore: '<table><tbody><tr><td>ab</td><td>cd</td><td>ef[]</td></tr></tbody></table>',
+                    contentBefore: '<table><tbody><tr style="height: 20px;"><td style="width: 20px;">ab</td><td>cd</td><td>ef[]</td></tr></tbody></table>',
                     stepFunction: async editor => triggerEvent(editor.editable, 'keydown', { key: 'Tab'}),
-                    contentAfter: '<table><tbody><tr><td>ab</td><td>cd</td><td>ef</td></tr><tr><td>[]<br></td><td><br></td><td><br></td></tr></tbody></table>',
+                    contentAfter: '<table><tbody><tr style="height: 20px;"><td style="width: 20px;">ab</td><td>cd</td><td>ef</td></tr><tr style="height: 20px;"><td>[]<br></td><td><br></td><td><br></td></tr></tbody></table>',
```

Comment:
So we can never have rows with "default" height that is no height specifically defined on them ?

Reply:
Not when we add a row because we want to preserve height proportions.